funlet-call-me/tests/funlet-call-me.test.js

```diff
@@ -56,64 +54,43 @@ const NO_FALLBACK_URL = '';
 const DEFAULT_FALLBACK_URL = NO_FALLBACK_URL;
 
 const DEFAULT_WHISPER_URL = '.?Whisper=true';
-const WHISPER_URL_WITH_CUSTOM_MESSAGE =
-  DEFAULT_WHISPER_URL + '&amp;Message=' + CUSTOM_MESSAGE_ENCODED;
+const WHISPER_URL_WITH_CUSTOM_MESSAGE = `${DEFAULT_WHISPER_URL}&amp;Message=${CUSTOM_MESSAGE_ENCODED}`;
 
 const XML_DECLARATION = '<?xml version="1.0" encoding="UTF-8"?>';
 
 const FULL_RESPONSE_CALL_ME_1_1 =
-  XML_DECLARATION +
-  '<Response>' +
-  '<Dial ' +
-  'action=".?Dial=true" ' +
-  'timeout="' +
-  DEFAULT_TIMEOUT +
-  '"' +
-  '>' +
-  '<Number url="' +
-  DEFAULT_WHISPER_URL +
-  '">' +
-  PHONE_NUMBER +
-  '</Number>' +
-  '</Dial>' +
-  '</Response>';
+  `${XML_DECLARATION}<Response>` +
```

Comment:
you can write multiline strings with the template literal you're using here. No need to start stop it on every line. Just at the start of the declaration and close it at the end